.gitignore

```diff
@@ -9,3 +9,4 @@ test/lib/
 test/pip-selfcheck.json
 node_modules
 /dist
+*.swp
```

Comment:
This feels like something that should go into your global gitignore for your workstation.